dicom-archive/get_dicom_info.pl

```diff
@@ -190,18 +329,47 @@ sub vector_cross_product {
     return @result;
 }
 
+=pod
+
+=head3 trim($input)
+
+Remove leading and trailing spaces from the $input variable
+
+INPUT: string to remove leading and trailing spaces from
+
+RESULTS: string without leading and trailing spaces
```

Comment:
RESULTS => RETURNS

Reply:
Oh and I know it's not part of your PR but this method should be removed: there's already a Perl built-in function that does exactly this: `chomp`